Renee-1102-Word-game_starter.py

```diff
@@ -0,0 +1,155 @@
+import random
+import string
+
+WORDLIST_FILENAME = "word_list.txt"
+
+def load_words():
+    """
+    Returns a list of valid words. Words are strings of lowercase letters.
+    
+    Depending on the size of the word list, this function may
+    take a while to finish.
+    """
+    print("Reading word_list file...")
+    # inFile: file
+    inFile = open(WORDLIST_FILENAME, 'r')
+    # line: string
+    line = inFile.readline()
+    # word_list: list of strings
+    word_list = line.split()
+    print(len(word_list), "words found")
+    return word_list
+
+def choose_word(word_list):
+    """
+    word_list (list): list of words (strings)
+    Returns a word from word_list at random
+    """
+    return random.choice(word_list)
+
+# end of helper code
+# -----------------------------------
+
+# Load the list of words into the variable word_list
+# so that it can be accessed from anywhere in the program
+word_list = load_words()
+
+def is_word_guessed(secret_word, letters_guessed):
+    '''
+    secret_word: string, the word the user is guessing
+    letters_guessed: list, what letters have been guessed so far
+    returns: boolean, True if all the letters of secret_word are in letters_guessed;
+      False otherwise
+    '''
+    # FILL IN YOUR CODE HERE...
+    pass
+  #secret_word = random(word_list)
+  #letters_guessed = input(range(list))
+  #list = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
+#      
+
+
+    for letter in secret_word:
+        if letter not in letters_guessed:
+          return False
+
+    return True
+
+### Testcases
+#secret_word = 'renee'
+#letters_guessed = ['e', 'f', 'r', 'n', 'o']
+#print(is_word_guessed(secret_word, letters_guessed))
+
+#print(is_word_guessed('apple', ['a', 'e', 'i', 'k', 'p', 'r', 's']))
+# print(is_word_guessed('durian', ['h', 'a', 'c', 'd', 'i', 'm', 'n', 'r', 't', 'u']))
+# print(is_word_guessed('pineapple', []))
+
+
+
+def get_guessed_word(secret_word, letters_guessed):
```

Comment:
print(get_guessed_word('apple', ['e', 'i', 'k', 'p', 'r', 's']))

ppe

but should be _ pp_ e